### DISCUSSION TITLE
Support reordering notebook index

### DISCUSSION THREAD

**Discussion by yixin0829** [tamlok/vnote#1708] [Ideas]
Hi is it possible for vnote to support reordering the notebook index by dragging? I noticed rn all the created files would go to the end in the index. 🙏

**Comment by tamlok** [comment]
Hi,

In beta.8, there is an eye icon at the top of the notebook panel to let you choose the view order.

![image](https://user-images.githubusercontent.com/5763157/109911349-df8ed680-7ce4-11eb-8df8-848ff5c4e606.png)

For manual sort, it will be supported in beta.9.

**Reply by yixin0829** [reply]
Thanks. looking forward to beta9!
